nuclio/utils.py

```diff
@@ -159,6 +159,11 @@ def normalize_name(name):
     return name.lower()
 
 
+def add_nuclio_suffix_to_name(name):
+    # This way we avoid import issues if the filename is the same as an existing Python library
```

Comment:
While this comment is true, it is irrelevant in this helper function (that simply adds a suffix to a string)